src/svgSprite.ts

```diff
@@ -1098,26 +1280,41 @@ export const svgSprite = (): string => `
     </symbol>
-
+  `,
+  'limited-account': `
     <symbol viewBox="0 0 32 32" id="svg-limited-account">
       <path d="M22.666 20.0002C21.7327 20.0002 20.9438 19.6779 20.2993 19.0335C19.6549 18.3891 19.3327 17.6002 19.3327 16.6668C19.3327 15.7335 19.6549 14.9446 20.2993 14.3002C20.9438 13.6557 21.7327 13.3335 22.666 13.3335C23.5993 13.3335 24.3882 13.6557 25.0327 14.3002C25.6771 14.9446 25.9993 15.7335 25.9993 16.6668C25.9993 17.6002 25.6771 18.3891 25.0327 19.0335C24.3882 19.6779 23.5993 20.0002 22.666 20.0002ZM17.3327 26.6668C16.9549 26.6668 16.6385 26.5388 16.3833 26.2828C16.1273 26.0277 15.9993 25.7113 15.9993 25.3335V24.8002C15.9993 24.2668 16.138 23.7726 16.4153 23.3175C16.6936 22.8615 17.0882 22.5335 17.5993 22.3335C18.3993 22.0002 19.2273 21.7504 20.0833 21.5842C20.9385 21.4171 21.7993 21.3335 22.666 21.3335C23.5327 21.3335 24.394 21.4171 25.25 21.5842C26.1051 21.7504 26.9327 22.0002 27.7327 22.3335C28.2438 22.5335 28.6385 22.8615 28.9167 23.3175C29.194 23.7726 29.3327 24.2668 29.3327 24.8002V25.3335C29.3327 25.7113 29.2047 26.0277 28.9487 26.2828C28.6936 26.5388 28.3771 26.6668 27.9993 26.6668H17.3327ZM13.3327 16.0002C11.866 16.0002 10.6105 15.4779 9.56602 14.4335C8.52157 13.3891 7.99935 12.1335 7.99935 10.6668C7.99935 9.20016 8.52157 7.94461 9.56602 6.90016C10.6105 5.85572 11.866 5.3335 13.3327 5.3335C14.7993 5.3335 16.0549 5.85572 17.0993 6.90016C18.1438 7.94461 18.666 9.20016 18.666 10.6668C18.666 12.1335 18.1438 13.3891 17.0993 14.4335C16.0549 15.4779 14.7993 16.0002 13.3327 16.0002ZM3.99935 26.6668C3.62157 26.6668 3.30513 26.5388 3.05002 26.2828C2.79402 26.0277 2.66602 25.7113 2.66602 25.3335V22.6668C2.66602 22.0891 2.86602 21.5171 3.26602 20.9508C3.66602 20.3837 4.2549 19.8891 5.03268 19.4668C6.23268 18.8224 7.54379 18.3059 8.96602 17.9175C10.3882 17.5282 11.8438 17.3335 13.3327 17.3335C13.9327 17.3335 14.5718 17.3779 15.25 17.4668C15.9273 17.5557 16.6216 17.6891 17.3327 17.8668V19.6002C17.0216 19.7113 16.7549 19.8224 16.5327 19.9335C16.3105 20.0446 16.0771 20.1668 15.8327 20.3002C15.4327 20.1891 15.0162 20.1113 14.5833 20.0668C14.1496 20.0224 13.7327 20.0002 13.3327 20.0002C12.0438 20.0002 10.7829 20.1557 9.55002 20.4668C8.31624 20.7779 7.13268 21.2224 5.99935 21.8002C5.77713 21.9113 5.61046 22.0668 5.49935 22.2668C5.38824 22.4668 5.33268 22.6891 5.33268 22.9335V24.0002H13.3327V26.6668H3.99935ZM13.3327 13.3335C14.066 13.3335 14.694 13.0722 15.2167 12.5495C15.7385 12.0277 15.9993 11.4002 15.9993 10.6668C15.9993 9.9335 15.7385 9.30594 15.2167 8.78416C14.694 8.2615 14.066 8.00016 13.3327 8.00016C12.5993 8.00016 11.9718 8.2615 11.45 8.78416C10.9273 9.30594 10.666 9.9335 10.666 10.6668C10.666 11.4002 10.9273 12.0277 11.45 12.5495C11.9718 13.0722 12.5993 13.3335 13.3327 13.3335Z"/>
     </symbol>
+  `,
+};
+
+export type Icons = keyof typeof icons;
+const iconKeys = Object.keys(icons) as Icons[];
+
+export const svgSprite = (): string => `
```

Comment:
What do you think about replacing the current implementation of svgSprite with the following code?
``export const svgSprite = (): string => {
  const iconDefs = Object.values(icons).join('');
  
  return `<svg><defs>${iconDefs}</defs></svg>`;
};``
This could slightly improve performance.

Reply:
@IhorNesterenko99 not sure which would be more performant although the current version is slightly more readable.

How about this:
```
export const svgSprite = (): string => {
const iconDefs = Object.values(icons).join('');

return `
<svg>
  <defs>
    ${iconDefs}
  </defs>
</svg>
`;
};
```

Reply:
@mbanting  It looks like your last code snippet aligns with my proposal. If there are no objections, we could implement it.